app/Repositories/PaymentReconcileRepository.php

```diff
@@ -29,13 +29,15 @@ public function storeOrCreate(array $data)
 
         $resp = PaymentService::logPayment($data, $model);
 
-        if(request()->has('comment')){
+        if (request()->has('comment')) {
             $resp->comment()->create(['comment' => request('comment'), 'user_id' => auth()->user()->id]);
         }
         $payment_type = PaymentType::where('id', $data["payment_type_id"])->first();
 
         if ($payment_type->type == PaymentType::REPAYMENTS) {
-            event(new RepaymentEvent($model));
+            if (env('SEND_REORDER_SMS')) {
```

Comment:
Don't check here, there are other listeners that could be listening to this `RepaymentEvent` event